src/test/java/org/reaktivity/nukleus/oauth/internal/OAuthRealmsTest.java

```diff
@@ -18,19 +18,24 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.reaktivity.specification.nukleus.oauth.internal.OAuthJwtKeys.RFC7515_ES256;
+import static org.reaktivity.specification.nukleus.oauth.internal.OAuthJwtKeys.RFC7515_RS256;
 
+import org.jose4j.jws.JsonWebSignature;
+import org.jose4j.jwt.JwtClaims;
 import org.junit.Test;
 
 public class OAuthRealmsTest
 {
+    private static final String[] EMPTY_STRING_ARRAY = new String[0];
 
     @Test
     public void shouldAddUpToMaximumRealms() throws Exception
     {
         OAuthRealms realms = new OAuthRealms();
         for (int i=0; i < Short.SIZE; i++)
         {
-            realms.add("realm" + i);
+            realms.resolve("realm" + i, EMPTY_STRING_ARRAY);
```

Comment:
Consider adding a `resolve` method to `OAuthRealms` that internally calls the 2 parameter `resolve` with the passed in `realmName` and `EMPTY_STRING_ARRAY`.
That will simplify these tests and eliminate the diff.